higher_health/models.py

```diff
@@ -17,26 +19,36 @@ class University(models.Model):
         (s.code, s.name) for s in pycountry.subdivisions.get(country_code="ZA")
     )
     name = models.CharField(max_length=100)
-    province = models.CharField(choices=PROVINCE_CHOICES, max_length=100)
+    province = models.CharField(choices=PROVINCE_CHOICES, max_length=100, blank=True)
+    sort_order = models.IntegerField(default=0, blank=True, db_index=True)
```

Comment:
I don't think we need database indexes on this fields, or the one for Campus. We're fetching the whole table every time, so the index will never be used.